stile/data_source_stile_manifest.go

```diff
@@ -86,10 +86,13 @@ func getBuildkiteArtifact(artifactName string, buildNumber string, pipeline stri
 		diags = append(diags, diag.Diagnostic{
 			Severity: diag.Error,
 			Summary:  "Unable to find list buildkite artifacts.",
-			Detail:   fmt.Sprintf("Attempted to read BFP Build %s", buildNumber),
+			Detail: fmt.Sprintf(
+				"Attempted to read BFP Build %s. This can mean the artifact does not exist or your token has insufficient permission to access it.",
+				buildNumber,
```

Comment:
```suggestion
				"Attempted to read BFP Build %s. This can mean the artifact does not exist or your Buildkite API token has insufficient permission to access it.",
				buildNumber,
```